src/libsync/propagateuploadng.cpp

```diff
@@ -376,7 +370,7 @@ void PropagateUploadFileNG::slotPutFinished()
         _item->_httpErrorCode = job->reply()->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
         QByteArray replyContent;
         QString errorString = job->errorStringParsingBody(&replyContent);
-        qDebug() << replyContent; // display the XML error in the debug
+        qCDebug(lcPropagateUpload) << replyContent; // display the XML error in the debug
```

Comment:
same as for v1, i think those errors should be displayed more easily as the log is important for people to find out what's going wrong